Makefile

```diff
@@ -1,3 +1,4 @@
+include mk/python.mk
```

Comment:
You should replace all occurrence of python invocation with wrapped `$(PYTHON)` for consistency and availability.